rumqttc/CHANGELOG.md

```diff
@@ -9,9 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Expose `EventLoop::clean` to allow triggering shutdown and subsequent storage of pending requests
+- Add support for `SEC1` encoded TLS keys (specifically focusing on elliptic curve cryptography (ECC)).
```

Comment:
we are adding support for other encoded keys ( supported by rustls pemfile ) as well right? can we make it something like that instead of just saying SEC1?

Reply:
I have changed it.